hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/services/ITestBlobOperation.java

```diff
@@ -165,31 +177,35 @@ public void testPutBlockWithNullData() throws Exception {
                 abfsConfiguration));
 
         String blockId = "block1";
-        byte[] data = null;
+        byte[] data = new byte[0];
```

Comment:
What is the change included as part of this test ?

Reply:
In this class' tests, there was calling of execute on AbfsRestOperation object (created inside the test). Now, if urls are created, correct endpoint selection would be required because now the uri change no longer happens at fs and store level. So, if instead use client's relevant method (which corresponds to the abfsRestOperation objects already getting created), test no longer need to maintain the uri conversion heuristic.